### PR TITLE
Added test cases for coci partial points

### PR DESCRIPTION
**Pull request by wesley-a-leung**
Test cases were added for the new COCI ContribModule to ensure that the fraction output format to stderr work as expected.

### CODE REVIEW

**Comment by kiritofeng** [comment]
Can you add a case where you print a ton of garbage to stderr, followed by a newline, then `partial 1/10` or something

**Comment by wesley-a-leung** [comment]
> Can you add a case where you print a ton of garbage to stderr, followed by a newline, then `partial 1/10` or something

I've added the case [here](https://github.com/DMOJ/judge-testsuite/blob/e0c56d6ae6292ad56a7a1e79630e667a09708e9d/coci_partial/tests/py_garbage_ie/ie.py).

**Comment by kiritofeng** [comment]
> 
> 
> > Can you add a case where you print a ton of garbage to stderr, followed by a newline, then `partial 1/10` or something
> 
> I've added the case [here](https://github.com/DMOJ/judge-testsuite/blob/e0c56d6ae6292ad56a7a1e79630e667a09708e9d/coci_partial/tests/py_garbage_ie/ie.py).

Intended behaviour for this is actually AC 1/10, which is why I had you add it (the code you presumably copy-pasted from my fix earlier today had an error in it)

**Comment by wesley-a-leung** [comment]
> > > Can you add a case where you print a ton of garbage to stderr, followed by a newline, then `partial 1/10` or something
> > 
> > 
> > I've added the case [here](https://github.com/DMOJ/judge-testsuite/blob/e0c56d6ae6292ad56a7a1e79630e667a09708e9d/coci_partial/tests/py_garbage_ie/ie.py).
> 
> Intended behaviour for this is actually AC 1/10, which is why I had you add it (the code you presumably copy-pasted from my fix earlier today had an error in it)

The intended behaviour has been fixed.

File: coci_partial/init.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+checker:
+  name: bridged
+  args:
+    files: checker.cpp
+    lang: CPP11
```
Comment (kiritofeng) [inline coci_partial/init.yml:5]: Running locally yielded
```
checker.cpp:8:53: note: ‘std::make_unique’ is only available from C++14 onwards
```
so perhaps you should switch to C++14

File: coci_partial/tests/py_garbage_ac/test.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+language: PY3
+time: 2
+memory: 65536
+source: ie.py
```
Comment (kiritofeng) [inline coci_partial/tests/py_garbage_ac/test.yml:4]: Source is now `ac.py`

**Pull request merged**